src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+function convertToNum(salary) {
+  return Number(salary.replace(/[^0-9.-]+/g, ''));
+}
+
+// Select the first <ul> element (or adjust the selector as needed)
+const list = document.querySelector('ul');
```

Comment:
Consider checking if `list` is not null before proceeding. If the `ul` element is not found, `list` will be null, and attempting to access `list.children` will cause a runtime error.